assets/javascripts/discourse/services/chat.js

```diff
@@ -225,6 +229,50 @@ export default Service.extend({
     });
   },
 
+  switchChannelUpOrDown(direction) {
+    const activeChannel = this.getActiveChannel();
```

Comment:
btw. could you change getActiveChannel to a native getter `get activeChannel() {`? 😃 

```suggestion
    const { activeChannel } = this;
```